code/src/components/Parent.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable default-case */
+/* eslint-disable no-unused-vars */
+/* eslint-disable eqeqeq */
+/* eslint-disable react/self-closing-comp */
+/* eslint-disable react/style-prop-object */
+/* eslint-disable no-nested-ternary */
+/* eslint-disable max-len */
+/* eslint-disable react-hooks/exhaustive-deps */
+import React from 'react';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components/macro';
+// Background images //
+import dream from 'assets/images/bed.jpg';
+import library from 'assets/images/library.jpg';
+import sunrise from 'assets/images/sunrise.jpeg';
+import cavernlight from 'assets/images/cavernlight.jpg';
+import cogs from 'assets/images/wheel.jpg';
+import colorfulcave from 'assets/images/colorfulcave.jpeg';
+import colorswirl from 'assets/images/colorswirl.jpg';
+import temple from 'assets/images/housetree.jpg';
+import archway from 'assets/images/archway.jpg'
+import library2 from 'assets/images/library2.jpeg'
+// mounted components
+import Start from './Start';
+import { InnerWrapper } from './reusableStyles';
+import Location from './Location';
+import Loading from './Loading';
+import Map from './Map';
+
+const Parent = () => {
+  const username = useSelector((store) => store.labyrinth.username)
+  const currentLocation = useSelector((store) => store.labyrinth)
+
+  const switchBackground = () => {
+    switch (currentLocation.coordinates) {
+      case '': return dream;
+      case '0,0': return archway;
+      case '0,1': return cogs;
+      case '0,2': return colorswirl;
+      case '0,3': return library2;
+      case '1,0': return cavernlight;
+      case '1,1': return colorfulcave;
+      case '1,3': return sunrise;
+    }
+  }
```

Comment:
Good job taking inspiration from this week's material and implementing the vanilla redux. 👍